docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -988,31 +1007,40 @@ thor: []
       </sect3>
 
       <sect3 id="FileAccessCosts">
-        <title>File Access Cost</title>
+        <title>File Access Costs</title>
+<para> For logical files, the costs calculated are based on two types of file access.  </para>
+<itemizedlist>
+<listitem>File Access Cost
+</listitem>     
+<listitem>File Cost at Rest
+</listitem>
+</itemizedlist>
+
 
         <para>File access costs are the costs of reading and writing to the
         files. Many storage planes do have a separate charge for data
-        operations. The cost of reading and writing to the file will be
+        operations. The cost of reading and writing to the file is
         included in the file access cost value. Any other cost associated with
         file operations (such as delete or copy) will not be tracked or
         included as part of file access cost at this time.</para>
-
+        
         <para>The file access cost displays as part of the cost field on the
         Logical File’s summary page in ECL Watch.</para>
 
         <para>The costs incurred by a workunit for accessing logical files is
         also recorded in the workunit’s statistics and attributes. The
         read/write cost is recorded at the activity record and cumulated at
-        the graph, the subgraph and the workflow scope level. The total file
-        access cost for a workunit is recorded with the workunit and displayed
-        in the summary page.</para>
+        the graph, the subgraph, and the workflow scope level.
+        Cost Tracking detects data that has been returned from the page cache and adjusts cost calculations appropriately to produce more accurate cost calculations. 
+        These file access costs for a workunit are recorded with the workunit and displayed
+        in the summary page and in the workunit’s metrics.</para>
 
-        <para>The new cost field is shown in the Logical File summary page. It
-        is the combined cost of storing and accessing the data.</para>
+      <sect4 id="FileCostAtRest"> <title>File Cost at Rest</title>
 
-        <para>The cost information is currently only generated for Thor and
-        hThor jobs.</para>
-      </sect3>
+        <para>The File Cost at Rest field is shown in the Logical File summary page. It
+        is the cost of storing the file without accessing the data. Only the storage costs associated with housing the file in the cloud. This value has been added to better differentiate between file storage costs. </para>
```

Comment:
"File Cost at Rest field" is the file storage costs.  I think the last line in the previous paragraph should be removed.  File Costs At Rest and File Access cost are both shown in the logical file's summary page.